addons/binding/pom.xml

```diff
@@ -110,6 +110,7 @@
 		<module>org.openhab.binding.urtsi</module>
 		<module>org.openhab.binding.vitotronic</module>
 		<module>org.openhab.binding.wifiled</module>
+		<module>org.openhab.binding.wifiled.test</module>
```

Comment:
Did you check whether it was ever included but removed? Anyone else some memory about this?

Reply:
It seemed to be missing from https://github.com/openhab/openhab2-addons/pull/2496 by @rvt which was merged yesterday.